<launch> 

  <group ns="small_excavator_1">
  
    <node pkg="move_excavator"
          name="find_rover"
          type="find_rover"
          output="screen">
    </node> >

    <node pkg="move_excavator"
          name="manipulation"
          type="manipulation"
          output="screen">
      <rosparam
        command="load"
        file="$(find move_excavator)/config/manipulation.yaml"
        subst_value="true"
      />
    </node>

  </group>

</launch>
